redis.conf

```diff
@@ -66,6 +66,8 @@ port 6379
 # will silently truncate it to the value of /proc/sys/net/core/somaxconn so
 # make sure to raise both the value of somaxconn and tcp_max_syn_backlog
 # in order to get the desired effect.
+# Linux will round this value up to the nearest power of two
+# using tcp-backlog + 1, so 511 becomes 512.
```

Comment:
```suggestion
```
POSIX says this is just a hint. Apparently some versions of Linux may round it to next power of 2 or MIN 16. But I don't think it's a guarantee so I don't think it needs to be specified.